### PR TITLE
Add properties to signed_transaction

### PR DESCRIPTION
**Pull request by myriamchah**
### Context 

Add properties to Metabase event signed_transaction

### Specs 

[WLT-4699](https://zerion-tech.atlassian.net/browse/WLT-4699) part 4

[WLT-4699]: https://zerion-tech.atlassian.net/browse/WLT-4699?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by zerts — APPROVED** [review]

File: src/shared/analytics/shared/addressActionToAnalytics.ts [hunk 1]
```diff
@@ -68,6 +70,10 @@ function getAssetName({ asset }: { asset: ActionAsset }) {
   return getFungibleAsset(asset)?.name;
 }
 
+function getAssetAddresses({ asset }: { asset: ActionAsset }) {
```
Comment (everdimension) [inline src/shared/analytics/shared/addressActionToAnalytics.ts:73]: why "addresses" if it returns just one?

File: src/shared/analytics/shared/addressActionToAnalytics.ts [hunk 2]
```diff
@@ -99,17 +105,27 @@ export function addressActionToAnalytics({
...
     asset_name_received: toMaybeArr(incoming?.map(getAssetName)),
+    asset_address_sent: toMaybeArr(outgoing?.map(getAssetAddresses)),
+    asset_address_received: toMaybeArr(incoming?.map(getAssetAddresses)),
   };
   if (quote) {
     const zerion_fee_percentage = quote.protocol_fee;
     const feeAmount = quote.protocol_fee_amount;
     const asset = incoming?.[0]?.asset;
+    const output_chain = quote.output_chain;
+    const transaction_zerion_fee_asset_id = getFungibleAsset(asset)?.id;
```
Comment (everdimension) [inline src/shared/analytics/shared/addressActionToAnalytics.ts:116]: I think the name implies that this should be id of asset that the fee is paid with, but we assign the id of the incoming asset instead?

**Review by everdimension — CHANGES_REQUESTED** [review]
See comments

File: src/shared/analytics/shared/addressActionToAnalytics.ts (hunk 1 above)
Comment (myriamchah) [reply src/shared/analytics/shared/addressActionToAnalytics.ts:73]: Oops right! Got misled 👀

File: src/shared/analytics/shared/addressActionToAnalytics.ts (hunk 2 above)
Comment (myriamchah) [reply src/shared/analytics/shared/addressActionToAnalytics.ts:116]: In what case can incoming asset and zerion fee asset be different?

Comment (myriamchah) [reply src/shared/analytics/shared/addressActionToAnalytics.ts:116]: The field is not required in the specs anymore (I validated with Anna) so I'll remove it rn, thank you!

**Review by everdimension — APPROVED** [review]

**Pull request merged**